### ISSUE TITLE
sidestake_check logic error?

### ISSUE THREAD

**Issue by HeyMerlin** [makeasnek/FindTheMag2#27]
If user answers 'no' to sidestaking, sidestake_check will still prompt for sidestake percentage with check_type = 'FOUNDATION'.

```
        while answer not in ['Y', 'N']:
            print('Error: Y or N not entered. Try again please :)')
            answer = input("")
        if answer == 'N':
            if check_type=='FOUNDATION':
                print('Ok no problem, it is your choice after all!')
            if check_type=='DEVELOPER':
                print('Ok no problem, it is your choice after all!')
                return
        print(message2)
        answer = input("")
```
The `return` should be moved out 1 level, or a return added to the first if statement.

**Comment by makeasnek** [comment]
I am working on the overhaul branch now and have added a fix for this. Many more commits to come in the following days.

**Comment by makeasnek** [comment]
Fixed in latest release v 3.0
